negotiation/diff.emu

```diff
@@ -360,23 +360,20 @@
     </emu-clause>
 
     <ins class="block">
-    <emu-clause id="sec-getstringorbooleanoption" aoid="GetStringOrBooleanOption">
-      <h1>GetStringOrBooleanOption ( _options_, _property_, _values_, _trueValue_, _falsyValue_, _fallback_ )</h1>
+    <emu-clause id="sec-getbooleanorstringnumberformatoption" aoid="GetBooleanOrStringNumberFormatOption">
+      <h1>GetBooleanOrStringNumberFormatOption ( _options_, _property_, _stringValues_, _fallback_ )</h1>
 
       <p>
-        The abstract operation GetStringOrBooleanOption extracts the value of the property named _property_ from the provided _options_ object. It returns either _trueValue_, _falsyValue_, _fallback_, or one of the elements of the List _values_. The following steps are taken:
+        The abstract operation GetBooleanOrStringNumberFormatOption extracts the value of the property named _property_ from the provided _options_ object. It returns either *true*, *false*, _fallback_, or one of the elements of the List _values_. The following steps are taken:
       </p>
 
-     <emu-alg>
+      <emu-alg>
         1. Let _value_ be ? Get(_options_, _property_).
         1. If _value_ is *undefined*, return _fallback_.
-        1. If _value_ is *true*, return _trueValue_.
-        1. Let _valueBoolean_ be ToBoolean(_value_).
-        1. If _valueBoolean_ is *false*, return _falsyValue_.
+        1. If _value_ is *true*, return *true*.
+        1. If ToBoolean(_value_) is *false*, return *false*.
         1. Let _value_ be ? ToString(_value_).
-        1. NOTE: For historical reasons, the strings *"true"* and *"false"* are treated the same as the _fallback_ value.
-        1. If _value_ is *"true"* or *"false"*, return _fallback_.
-        1. If _values_ does not contain an element equal to _value_, throw a *RangeError* exception.
+        1. If _stringValues_ does not contain an element equal to _value_, throw a *RangeError* exception.
```

Comment:
```suggestion
        1. If _stringValues_ does not contain _value_, throw a *RangeError* exception.
```

Reply:
"an element equal to" comes from other AOs in this file.

Reply:
Are you planning to fix them all together as part of an "adopt current conventions" PR?

Reply:
PTAL